nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeoctl.bat

```diff
@@ -87,9 +87,11 @@ FOR /F "usebackq eol=# tokens=1,* delims==" %%A in ("%NUXEO_CONF%") do (
   if "%%A" == "nuxeo.log.dir" set LOCAL_NUXEO_LOG_DIR=%%B
   if "%%A" == "nuxeo.tmp.dir" set LOCAL_NUXEO_TMP_DIR=%%B
   if "%%A" == "JAVA_OPTS" (
-    set __JAVA_OPTS=%%B
```

Comment:
why the renaming of `__JAVA_OPTS` to `NEW_OPTS`?

Reply:
Because it's confusing to see:

- `JAVA_OPTS` (var)
- `LOCAL_JAVA_OPTS` (var)
- `__JAVA_OPTS` (var)
- `JAVA_OPTS` (configuration constant, part A)
- `$JAVA_OPTS` (configuration constant, part B)

I believe using `NEW_OPTS` is easier to read. => I can remove it.

Reply:
I agree on the principle but it is there for 5 years and I don't feel it worth to change now.
Note also that there are BATCH naming conventions (that I'm not very aware of). For instance, a global variable  that is required only for the current execution should start with an underscore to ensure that variable name conflicts will not occur.